lib/hyrax/configuration.rb

```diff
@@ -348,6 +348,32 @@ def cache_path
       @cache_path ||= ->() { Rails.root + 'tmp' + 'uploads' + 'cache' }
     end
 
+    attr_writer :iiif_image_server
+    def iiif_image_server?
+      return @iiif_image_server unless @iiif_image_server.nil?
+      @iiif_image_server = false
+    end
+
+    attr_writer :iiif_image_url_builder
+    def iiif_image_url_builder
```

Comment:
I think adding documentation here would be good.  Also. should we add this to the configuration template?

Reply:
I see this is already in the template, so just docs here.

Reply:
+1 to more doc here.

Reply:
I thought I did add the five new configs to the template: https://github.com/samvera/hyrax/pull/2311/files#diff-e3d30f16d40cc1c49bbe4d548e3f4d23

Reply:
@jcoyne @jenlindner k